api/config/cdr_config_local.json

```diff
@@ -147,6 +146,29 @@
       "hasSurveyConductData": true,
       "accessTier": "registered",
       "tanagraEnabled": true
+    },
+    {
+      "cdrVersionId": 10,
+      "isDefault": true,
+      "name": "Synthetic Dataset in the Controlled Tier v8 (Echo)",
+      "bigqueryProject": "fc-aou-cdr-synth-test-2",
+      "bigqueryDataset": "SC2023Q3R3",
+      "wgsBigqueryDataset": "echo_controls",
+      "wgsFilterSetName": "echo-controls",
+      "creationTime": "2024-11-04 19:58:48Z",
+      "numParticipants": 234525,
+      "cdrDbName": "cdr",
```

Comment:
mostly copied from test except this value - it must be "cdr" because of how Local works